lib/Models/Catalog/CatalogReferences/ThreddsItemReference.ts

```diff
@@ -121,7 +121,7 @@ export default class ThreddsItemReference extends UrlMixin(
     );
     if (model === undefined) return;
     this.setThreddsStrata(model);
-    previousTarget = model;
+    _previousTarget = model;
```

Comment:
What do these assignments do, is `previousTarget` some kind of return-value type so the caller gets a hold of a reference to the model?

Reply:
That's my understanding, it's updating the reference. Although looking at it's call in `ReferenceMixin` that reference doesn't appear to be further used. @na9da do you have insight into this?

Reply:
I think that line has no effect and can be removed. There's [couple more](https://github.com/TerriaJS/terriajs/blob/a21ecf7a75a66d328c5340fc24522f7c56d17c2d/lib/Models/Catalog/Ckan/CkanItemReference.ts#L391) [of these](https://github.com/TerriaJS/terriajs/blob/a21ecf7a75a66d328c5340fc24522f7c56d17c2d/lib/Models/Catalog/Esri/ArcGisPortalItemReference.ts#L353).